### ISSUE TITLE
Password/Encryption for Manu

### ISSUE THREAD

**Issue by linuxtopia** [olivierkes/manuskript#605]
Hey guys,

Could be added the **encryption/password** protection support to open projects? Would be nice feature.

Thanks

**Comment by worstje** [comment]
Thank you for the suggestion.

Personally, I think encryption and password protection is outside of the scope of the Manuskript project at this time. Besides the fact that 'security' is not something that is easy nor trivial to implement - especially in the Manuskript codebase! - and would likely cost more time of the volunteers working on the project than it is worth, there are also plenty of alternatives available when it comes to 'securing files'.

Have a look into filesystem-level encryption such as BitLocker (windows) or fscrypt (linux). Those store files in such a way that they are inaccessible if you aren't logged in to the machine, which is essentially the password and encryption protection you are looking for at a far more tested level than Manuskript could ever provide.

Of course, do note that these files will be accessible to anyone if you leave your computer unattended without logging out. But if you make that mistake, you would likely make the same sort of mistake if Manuskript implemented such an encryption feature, gaining you nothing.

Additionally, perhaps cloud services could offer you the desired 'gating' to the file that you desire. From your post alone, it is kind of hard to know exactly what problem you are trying to solve by trying to add more security to your projects, but I think the best security would probably be obtained from adjusting your workflow to be more security conscious of the scenarios you fear.

**Comment by TheJackiMonster** [comment]
I don't think this is necessarily out of scope since projects can be saved as one file which is just a .zip file. So using the encryption feature of zip could implement this request without too much effort. But this would only work when using this one-file structure.
